spec/support/grpc/temporal/api/workflowservice/v1/service_services_pb.rb

```diff
@@ -0,0 +1,259 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Hrmm, this approach inspired me to remove the gRPC dependency in Python - https://github.com/temporalio/sdk-python/issues/56. Thanks!

Reply:
Brilliant! 🙌